problems/sortedClassesBySize.js

```diff
@@ -11,7 +11,13 @@
 * returns: ["B", "C", "A"]
 */
 
-function sortClassesBySize() {
+const sortClassesBySize = (arr) => {
+    arr.sort((aclass, bclass) => {
+        return aclass.studentCount - bclass.studentCount;
+    });
+    return arr.map((aclass) => {
+        return arr.aclass;
+    });
```

Comment:
very close, what you have to remember when using the map method is that the parameter that is passed `aclass` becomes each element of the array per iteration. In this case each element was an object, so once we have the object we would need to access one of the object's keys to get the value out of it. For this question you forgot to access the `subject` key.